src/Firebase/firebase.js

```diff
@@ -121,7 +126,57 @@ export const doCreateProject = (obj) => {
       console.log(error);
     });
   });
-  });
+  }, (error) => {alert("Something went wrong\n"+ error); console.log("something went wrong")})
+  .catch(error => {alert("Something went wrong\n"+ error); console.log("something went wrong")});}
+
+  //in the absence of image
+  else{
+    let url = defaultImage
+    let user = firebase.auth().currentUser;
+    if (!user) {
+      alert("Please login to add a project");
+      return;
+    }
+    let title = obj.title;
+    let desc = obj.desc;
+    let links = obj.links;
+    let contactNo = obj.contactNo;
+    let githubLink = obj.githubLink;
+    let tags = obj.tags;
+    let teamMembers = obj.teamMembers;
+    let photo = url;
+    let uid = user.uid;
+    let leaderName = user.displayName;
```

Comment:
We can skip all these variable declarations and instead just specify them in the object itself right?